### PR TITLE
add date to create task

### PR DESCRIPTION
**Pull request by patrykbrodziak1**

### CODE REVIEW

**Review by kubaczerwinski77 — CHANGES_REQUESTED** [review]
The only suggestion is to change function name, because now we update all task properties - not only priority or status.

File: server/controllers/task.js [hunk 1]
```diff
@@ -3,22 +3,31 @@ import pool from "../db/setup.js";
 export default class TaskController {
   static createTask = async (req) => {
     await pool.query(
-      `INSERT INTO tasks (title, description, user_id, priority, status) VALUES ($1, $2, $3, $4, $5);`,
+      `INSERT INTO tasks (title, description, user_id, priority, status, due_date) VALUES ($1, $2, $3, $4, $5, $7);`,
```
Comment (pastepi) [inline server/controllers/task.js]: Any reason for `$7` and not `$6` here?

**Review by pastepi — CHANGES_REQUESTED** [review]

File: server/controllers/task.js (hunk 1 above)
Comment (patrykbrodziak1) [reply server/controllers/task.js]: no, missclick ofc

File: server/controllers/task.js [hunk 2]
```diff
@@ -3,22 +3,31 @@ import pool from "../db/setup.js";
...
         req.body.priority,
         req.body.status,
+        req.body.dueDate,
       ],
     );
     return { message: "Task created" };
   };
 
-  static changeTaskStatusOrPriority = async (req) => {
+  static changeTaskInfo = async (req) => {
```
Comment (pastepi) [inline server/controllers/task.js:19]: We need to change the method name and endpoint url in the tasks router as well.

**Review by pastepi — APPROVED** [review]

File: server/controllers/task.js (hunk 2 above)
Comment (patrykbrodziak1) [reply server/controllers/task.js:19]: I think it's okay. what endpoint name do you suggest?

Comment (patrykbrodziak1) [reply server/controllers/task.js:19]: image

**Review by pastepi — APPROVED** [review]

**Review by kubaczerwinski77 — APPROVED** [review]

**Pull request merged**